src/main/java/com/hazelcast/commandline/ProcessExecutor.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.hazelcast.commandline;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Handler for OS level process operations.
```

Comment:
Could you explain better what this class is about? After reading javadoc + code, I still don't know why we need it.

Reply:
I created it to separate process execution operation from the command line implementations. It is also helpful on tests.